src/plugins/workspace/public/components/workspace_menu/workspace_menu.scss

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.custom-title {
+  text-transform: none !important;
+  font-size: 15px !important;
+}
+
+.text-ellipsis {
```

Comment:
Actually there is a ellipsis class in OUI. Maybe you can try take a look

Reply:
<OuiExpression> seems invalid.

Reply:
`<OuiExpression
        onClick={() => {}}
        description={description}
        value={value}
        textWrap="truncate"
      />` is not apply to our scenario.

Reply:
https://oui.opensearch.org/1.8/#/utilities/css-utility-classes 

oui-textTruncate

Reply:
Updated